Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -548,7 +561,8 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                             priority: 100,
                             multiple: this.get('multiple'),
                             loadContent: this.get('loadContent'),
-                            isAlreadySelected: Y.bind(this._isAlreadySelected, this)
+                            isAlreadySelected: Y.bind(this._isAlreadySelected, this),
+                            startingLocationId: this.get('startingLocationId')
```

Comment:
this is not unit tested. Also given the `startingLocationId` was added in the method base view, the `eZ.UniversalDiscoverySearchView` instance below should also receive that parameter (even if for now, it does not do anything with it).